Lib/compositor/logger.py

```diff
@@ -16,7 +16,10 @@
 The returned log is in XML format.
 """
 
-from cStringIO import StringIO
+try:
+    from io import StringIO
```

Comment:
fontTools.misc.py23 has an abstracted StringIO, you might want to just use that.